bluepyopt/ephys/responses.py

```diff
@@ -84,3 +84,37 @@ def plot(self, axes):
             self.response['voltage'],
             label='%s' %
             self.name)
+
+
+class TimeLFPResponse(Response):
+
+    """Response to stimulus"""
```

Comment:
Not obvious what the difference is with the class above

Reply:
semantically, the `TimeVoltageResponese` is a single intracellulare voltage traace, the `TimeLFPResponse` is extracellular anc can be multichannel. The implementation is indeed the same (so we could inherit...)

Reply:
Something like ?
```
class TimeLFPResponse(TimeVoltageResponse):

    """Response to stimulus"""

    def __init__(self, name, time=None, lfp=None):
        """Constructor
        Args:
            name (str): name of this object
            time (list of floats): time series
            lfp (list of floats): voltage series
        """

        super(TimeLFPResponse, self).__init__(name, time=time, voltage=None)
        self.response["voltage"] = lfp

    def plot(self, axes):
        raise NotImplementedError

```